### PR TITLE
feat(user_manual): update screenshots and feature description (Talk 18)

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Update screenshots as per current stable28 appearance
* Add description to the features provided with stable28 release (some has been added before that)

### 🖼️ Screenshots

image
image
image

See also updated screenshots in files diff

### CODE REVIEW

File: user_manual/talk/images/share-screen-with-camera.png
Comment (nickvergessen) [inline user_manual/talk/images/share-screen-with-camera.png:1]: That's quite an unlikely fake "camera" image :see_no_evil:

**Review by nickvergessen — APPROVED** [review]
Looks okay otherwise

File: user_manual/talk/talk_basics.rst [hunk 1]
```diff
@@ -237,7 +237,9 @@ Starting a screen share
 -----------------------
 
 You can click the monitor icon on your video stream to share your screen. Depending on your browser, you will get the option to share a monitor, an application window or a single browser tab.
+If video from your camera is also available, other participants will see it in a small presenter view next to the screen share.
```
Comment (DorraJaouad) [inline user_manual/talk/talk_basics.rst:240]: Should we mention also that you can hide it and open it again from a button ?

File: user_manual/talk/join_a_call_or_chat_as_guest.rst [hunk 2]
```diff
@@ -9,11 +9,11 @@ You can find out more about Nextcloud Talk `on our website <https://nextcloud.co
 Joining a chat
 --------------
 
-If you received a link to a chat conversation, you can open this in your browser to join the chat.
+If you received a link to a chat conversation, you can open this in your browser to join the chat. You will be prompted to enter your name before joining a chat.
```
Comment (DorraJaouad) [inline user_manual/talk/join_a_call_or_chat_as_guest.rst:12]:
```suggestion
If you received a link to a chat conversation, you can open this latter in your browser to join the chat. You will be prompted to enter your name before joining a chat.
```

or "open it"

File: user_manual/talk/talk_basics.rst (hunk 1 above)
Comment (Antreesy) [reply user_manual/talk/talk_basics.rst:240]: Other participants can do it, so I think, not related to a user who's reading this manual

File: user_manual/talk/images/share-screen-with-camera.png
Comment (Antreesy) [reply user_manual/talk/images/share-screen-with-camera.png:1]: OBS is broken for me, can't setup a virtual camera.. But looks similare to the real output, so I'd keep it

**Review by DorraJaouad — APPROVED** [review]
Clear and understandable.

**Pull request merged**